api/authenticators/okta/okta_auth.go

```diff
@@ -83,14 +83,14 @@ func (o *oktaAuthClient) do(ctx context.Context, method, path string, data, resu
 
 	b, err := json.Marshal(data)
 	if err != nil {
-		return
+		return nil, WrapError(err, ErrOktaBadRequest)
```

Comment:
Because this function has a return signature with aliases in:

````go
func (*oktaAuthClient) do(...) (response *http.Response, err error)
```

We don't need to explicitly return two values every time. The following would suffice:

````go
if err != nil {
    err = WrapError(err, ErrOktaBadRequest)
    return
}
````